### ISSUE TITLE
When building from head, rules do not update without a shutdown of the server.

### ISSUE THREAD

**Issue by restingbull** [bazel-contrib/rules_kotlin#631]
This is due to bazel being unable to determine that changes in the local_repository directory affect the archive_repository.

**Comment by restingbull** [comment]
@nkoroste -- can you verify this works? It looks like it does locally.

**Comment by nkoroste** [comment]
No, I don't think it worked, I still have to `--expunge` in order for changes in rules_kotlin to take effect in the client repo that uses them
